kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/util/LinuxFileUtil.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2011, 2016 Eurotech and/or its affiliates
+ * Copyright (c) 2011, 2017 Eurotech and/or its affiliates
```

Comment:
Well, the method is still good to have. I believe it may as well remain where it is. But if you wish, I can move it.